src/test/java/sc/fiji/labkit/ui/plugin/CalculateProbabilityMapWithLabkitIJ1PluginTest.java

```diff
@@ -28,45 +28,47 @@
  */
 package sc.fiji.labkit.ui.plugin;
 
-import static org.junit.Assert.assertTrue;
-
-import java.io.File;
-import java.io.IOException;
-
 import ij.IJ;
 import ij.ImagePlus;
 import ij.macro.Interpreter;
 import net.imglib2.img.VirtualStackAdapter;
 import net.imglib2.test.ImgLib2Assert;
 import org.junit.Test;
-import sc.fiji.labkit.pixel_classification.utils.SingletonContext;
+import org.scijava.Context;
 import sc.fiji.labkit.ui.utils.TestResources;
 
+import java.io.File;
+import java.io.IOException;
+
+import static org.junit.Assert.assertTrue;
+
 public class CalculateProbabilityMapWithLabkitIJ1PluginTest {
 
 	@Test
 	public void test() throws IOException {
-		SingletonContext.getInstance();
-		String inputImage = TestResources.fullPath("/blobs.tif");
-		String blobsModel = TestResources.fullPath("/blobs.classifier");
-		String source = TestResources.fullPath("/blobs_probability_map.tif");
-		File outputImage = File.createTempFile("labkit-segmentation-test", ".tif");
-		String macroTemplate = "close('*');\n" +
-			"open('INPUT_TIF');\n" +
-			"run('Calculate Probability Map With Labkit (IJ1)', 'segmenter_file=SEGMENTER_FILE use_gpu=false');\n" +
-			"selectImage('probability map for blobs.tif');\n" +
-			"saveAs('Tiff', 'OUTPUT_TIF');\n" +
-			"close('*');\n";
-		String macro = macroTemplate
-			.replace('\'', '"')
-			.replace("INPUT_TIF", inputImage)
-			.replace("SEGMENTER_FILE", blobsModel)
-			.replace("OUTPUT_TIF", outputImage.getAbsolutePath());
-		new Interpreter().run(macro);
-		assertTrue(outputImage.exists());
-		ImagePlus expected = IJ.openImage(source);
-		ImagePlus result = IJ.openImage(outputImage.getAbsolutePath());
-		ImgLib2Assert.assertImageEquals(VirtualStackAdapter.wrap(expected),
-			VirtualStackAdapter.wrap(result), Object::equals);
+		try (Context context = new Context()) {
+			String inputImage = TestResources.fullPath("/blobs.tif");
+			String blobsModel = TestResources.fullPath("/blobs.classifier");
+			String source = TestResources.fullPath("/blobs_probability_map.tif");
+			File outputImage = File.createTempFile("labkit-segmentation-test", ".tif");
+			String macroTemplate = "close('*');\n" +
+				"open('INPUT_TIF');\n" +
+				"run('Calculate Probability Map With Labkit (IJ1)', 'segmenter_file=SEGMENTER_FILE use_gpu=false');\n" +
+				"selectImage('probability map for blobs.tif');\n" +
+				"saveAs('Tiff', 'OUTPUT_TIF');\n" +
+				"close('*');\n";
+			String macro = macroTemplate
+				.replace('\'', '"')
+				.replace("INPUT_TIF", inputImage)
+				.replace("SEGMENTER_FILE", blobsModel)
+				.replace("OUTPUT_TIF", outputImage.getAbsolutePath().replaceAll("\\\\", "\\\\\\\\"));
```

Comment:
I wonder if using apache-commons `StringEscapteUtils.escapeJava(text)` would be the even more correct solution here. But I cannot test it, and also wonder why the code  replaces double backslashes. Windows would use single backslashes in paths right?
```suggestion
				.replace("OUTPUT_TIF", StringEscapeUtils.escapeJava(outputImage.getAbsolutePath()));
```
Anyway, this is the test code, not the production code. I'm happy if it runs reliably on Windows, Linux and Mac.

Reply:
> wonder why the code replaces double backslashes.

Yeah, it's tricky. I should have written a comment about it, sorry... I have now pushed a commit that does so. ctrueden/labkit-ui@b82ab70b00194cbb8a0b8f2dfcb761c63b1a912f